integration_tests/e2e/cellConversion/index.cy.ts

```diff
@@ -565,6 +565,30 @@ context('Cell conversion', () => {
           changeCellCapacityPage.continueButton().click()
         })
 
+        it('can display Error validations previous selections after deselection when converting non-residential to residential cell', () => {
+          const confirmPage = Page.verifyOnPage(CellConversionConfirmPage)
+          confirmPage.backLink().click()
+          const changeCellCapacityPage = Page.verifyOnPage(CellConversionSetCellCapacityPage)
+          changeCellCapacityPage.backLink().click()
+          const setCellConversionSetCellTypePage = Page.verifyOnPage(CellConversionSetCellTypePage)
+          setCellConversionSetCellTypePage.cellTypeCheckboxItem('ACCESSIBLE_CELL').uncheck()
+          setCellConversionSetCellTypePage.cellTypeCheckboxItem('BIOHAZARD_DIRTY_PROTEST').uncheck()
+          setCellConversionSetCellTypePage.cellTypeCheckboxItem('CONSTANT_SUPERVISION').uncheck()
+          Page.verifyOnPage(CellConversionSetCellTypePage)
+          setCellConversionSetCellTypePage.continueButton().click()
+          Page.verifyOnPage(CellConversionSetCellTypePage)
+          setCellConversionSetCellTypePage.backLink().click()
+          const specificCellTypePage = Page.verifyOnPage(CellConversionSpecificCellTypePage)
+          specificCellTypePage.yesRadioButton().click()
+          specificCellTypePage.backLink().click()
+          const usedForPage = Page.verifyOnPage(CellConversionUsedForPage)
+          usedForPage.usedForCheckboxItem('CLOSE_SUPERVISION_CENTRE').uncheck()
+          usedForPage.usedForCheckboxItem('SUB_MISUSE_DRUG_RECOVERY').uncheck()
+          usedForPage.usedForCheckboxItem('FIRST_NIGHT_CENTRE').uncheck()
+          usedForPage.continueButton().click()
+          Page.verifyOnPage(CellConversionUsedForPage)
+        })
```

Comment:
We don't need to start at the confirmation page come all the way back from there. The scenario in the ticket starts from  the beginning of the journey, then goes through the steps up to the set capacity page, then clicks back from there to the used for page, then clears the checkboxes and submits. You can put a test this block on line 138 of this file:
```
describe('used for page', () => {
```
The steps of the test should be the same as those described in the ticket:

1. On the "Used For" or "Cell Type" screen, select one or more options.
2. Navigate to the next screen.
3. Return to the previous screen ("Used For" or "Cell Type").
4. Deselect the previously selected options so that no options remain selected.
5. Attempt to proceed to the next screen again.
6. Observe results (validation triggered but previously selected checkboxes checked again)

And then also you might want to add a similar test for the set cell types page.